app/components/footer.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link } from "@remix-run/react";
+import PrivacyPolicy from "~/routes/privacy_policy";
+
+import ReactDenverLogo from "../assets/react_denver_logo.svg";
+import YouTube from "../assets/YouTube.svg";
+import Discord from "../assets/Discord.svg";
+import GitHub from "../assets/GitHub.svg";
+import Twitter from "../assets/Twitter.svg";
+
+export default function Footer() {
+  return (
+    <div
+      style={{
+        height: "250px",
+        backgroundColor: "#A379C9",
+        display: "flex",
+        alignItems: "center",
+        paddingLeft: "5%",
+      }}
+    >
+      <img style={{ maxHeight: "50px" }} src={ReactDenverLogo} />
+      <a href="">
+        <img style={{ maxHeight: "50px" }} src={Discord} />
+      </a>
+      <a href="https://github.com/reactdenver">
+        <img style={{ maxHeight: "50px" }} src={GitHub} />
+      </a>
+      <a href="https://mobile.twitter.com/ReactDenver">
+        <img style={{ maxHeight: "50px" }} src={Twitter} />
+      </a>
+      <a href="https://www.youtube.com/c/ReactDenver">
+        <img style={{ maxHeight: "50px" }} src={YouTube} />
+      </a>
+      <a href="">Netlify</a>
+      <Link
+        style={{
+          padding: "0 5px 0 10px",
+          color: "#666666",
+          textDecoration: "none",
+        }}
+        to="/privacy_policy"
+      >
```

Comment:
We could get better styles out of Charka Link here - https://chakra-ui.com/docs/components/link

All if Charka's components have a prop `as` that allows you to modify the parent component in fun ways. For instance you can use the Charka Link with a Remix Link in the as to get all the goodness out of remix's link, like the to prop.

```jsx
import {Link as RouterLink} from "@remix-run/react"
import {Link} from "@chakra-ui/react"

return (
  <Link as={RouterLink} p={[0, 1, 0, 2.5]} to="/privacy_policy">Privacy Policy</Link>
)
```

Padding, margins etc can all be found - https://chakra-ui.com/docs/styled-system/style-props#margin-and-padding

Basically in my example above I have `p` == `padding` and I pass an array of the 4 values for padding. There's also `px` which is just the left, right and `py` which is top, bottom. But there is also `pt, pr, pb, pl` should you wish to go graniumar, such as overwriting just the right padding for example.